governance/votes/Executive vote - July 25, 2024.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Template - [Executive Vote] LITE-PSM-USDC-A Phase 1 Setup - July 25, 2024
```

Comment:
I'd like to propose to always use the "action" form across all titles and summary sections. This will make it easier to standardize. As a result, I'd formulate it as "Set up the Phase 1 of LITE-PSM-USDC-A"

Reply:
We strongly disagree. You do not need "action" form in the title, it adds clutter and length without adding additional information and breaking a convention dating back years. It is fine for the summary section but leave it out of the titles.

Please bring up changes like this at a cross-team meeting like the one later today for further discussion. They should not be made in "live" PRs.

Reply:
Ok, not a problem. The reason for suggesting this was to make it easier to explain and standardize in the instructions for drafting the Executive copy. I'm happy to have it either way, as long as we are consistent, which we have not been in the past.